README.md

```diff
@@ -1,4 +1,4 @@
-# Mattermost Operator for Kubernetes ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-operator/master.svg) ![Alpha](https://img.shields.io/badge/alfa-in%20progress-yellow.svg)
+# Mattermost Operator for Kubernetes ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-operator/master.svg) ![Alpha](https://img.shields.io/badge/alfa-in%20progress-yellow.svg) [![Community Server](https://img.shields.io/badge/Mattermost_Community-cloud_channel-blue.svg)](https://community.mattermost.com/core/channels/cloud)
 Mattermost Operator for Kubernetes is under construction.
```

Comment:
Can we merge this sentence and the `Status` section below? Seems redundant to have both.